kansha/card/comp.py

```diff
@@ -187,6 +122,20 @@ def move_card(self, card_index, column):
         column.data.cards.append(data_card)
         self.column = column
 
+    def card_dropped(self, request, response):
+        """
+        Dropped on new date (calendar view).
+        """
+        start = dateutil.parser.parse(request.GET['start']).date()
+        for __, extension in self.card_extensions:
+            extension().new_card_position(start)
+
+    ################################
+    # Feature methods, persistency #
+    ################################
+
+    # Members
+
     def get_authorized_users(self):
```

Comment:
Misleading name: not the users than are authorized to manage the card but "the users authorized to be added on this card"